stevecalla/romcom
# Feature/fix saved cover issue

## Is this a fix or a feature?
Fix
## What is the change?
Fixed save cover duplicates when view cover button clicked
## What does it fix?
See above.
## Where should the reviewer start?
Test the functionality for view cover button please.
## How should this be tested?
Click on view cover button = result equals stored default cover.
Click back to home, click on random cover, click on save cover. Do it three times.
Click on view cover button. It should display the default plus three random covers.
Click on view cover button. Ensure the result is not additional covers.
